app/controllers/content_items_controller.rb

```diff
@@ -97,6 +97,20 @@ def load_taxonomy_navigation
                                .yield_self { |document_collections| document_collections || [] }
                                .select { |document_collection| document_collection['document_type'] == 'document_collection' }
                                .map { |document_collection| document_collection.values_at('base_path', 'title') }
+
+      @banner_items = @content_item
+                        .content_item
+                        .dig('links', 'part_of_step_navs')
+                        .yield_self { |part_of_step_navs| part_of_step_navs || [] }
+                        .select { |step_by_step_nav| step_by_step_nav['document_type'] == 'step_by_step_nav' }
```

Comment:
`part_of_step_navs` should only contain `step_by_step_nav` documents so we can remove this line if we want.